sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -1255,7 +1255,9 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     if (
       typeof filter !== "boolean" &&
       typeof filter !== "string" &&
-      !correlationProperties.some((validProperty) => filter.hasOwnProperty(validProperty))
+      !correlationProperties.some((validProperty) =>
```

Comment:
We can delete this method completely.
"addRule" isn't being used anymore.

Reply:
I'd prefer to do it in a separate PR so that when we do a git blame in the future we get the right reason for deleting than one that would say "fixing linting errors"